Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1841,7 +1851,7 @@ public void TestAsyncLoginTimeout()
         {
             using (var conn = new MockSnowflakeDbConnection())
             {
-                int timeoutSec = 5;
+                int timeoutSec = 15;
```

Comment:
is it for purpose or maybe it was a temporary change for testing?

Reply:
I've revert